pkg/api/addons.go

```diff
@@ -97,9 +97,9 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 			{
 				Name:           DefaultBlobfuseFlexVolumeAddonName,
 				CPURequests:    "50m",
-				MemoryRequests: "10Mi",
+				MemoryRequests: "100Mi",
 				CPULimits:      "50m",
-				MemoryLimits:   "10Mi",
+				MemoryLimits:   "100Mi",
```

Comment:
does this belong in this PR? maybe we should change the other addons in a separate PR

Reply:
Yes, as-is they were not schedulable on moby + N series clusters.